### PR TITLE
test: add github action

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
test: add github action

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by feiskyer** [comment]
seems the Go version is too new?

```
Successfully setup go version ^1.13
go version go1.15 linux/amd64
```

**Comment by andyzhangx** [comment]
@mayankshah1607 could you fix this? thanks.

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    - name: Run build test
+      run: |
+        export PATH=$PATH:$HOME/.local/bin
+        make
+
+    - name: Make container test
+      run: |
+        export PATH=$PATH:$HOME/.local/bin
+        make container
```
Comment (mayankshah1607) [inline .github/workflows/ubuntu.yml:36]: @andyzhangx we have this test in travis CI as well as github Action. This test seems to be failing in travis due to permission issues. Shall we remove travis? It seems redundant.

Comment (andyzhangx) [reply .github/workflows/ubuntu.yml:36]: point is travis originally works, I still want to keep travis test running for a period of time, thanks.
And also now github action failed, need to fix that first.

File: test/sanity/run-test.sh [hunk 2]
```diff
@@ -16,13 +16,32 @@
...
+
+install_csi_sanity_bin
+
+readonly endpoint='unix:///tmp/csi.sock'
+nodeid='CSINode'
 if [[ "$#" -gt 0 ]] && [[ -n "$1" ]]; then
   nodeid="$1"
 fi
 
-_output/blobplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
```
Comment (andyzhangx) [inline test/sanity/run-test.sh:25]: it's blobplugin

File: test/sanity/run-test.sh [hunk 3]
```diff
@@ -16,13 +16,32 @@
...
+_output/smbplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
-echo "Begin to run sanity test..."
```
Comment (andyzhangx) [inline test/sanity/run-test.sh:27]: pls keep the original test cases

**Comment by mayankshah1607** [comment]
@andyzhangx It is fixed 👍 
There were issues due to fake mounter in the unit tests, some bugs in the scripts for sanity tests and some minor nits. Please take a look, thanks

**Review by ZeroMagic — APPROVED** [review]
/lgtm

**Pull request merged**